### PR TITLE
Add documentation for new array handling mechanisms.

### PR DESCRIPTION
**Pull request by schveiguy**
Followup to #2247 to document new behaviors. No code changes here, just ddoc changes.

### CODE REVIEW

**Comment by schveiguy** [comment]
ping @s-ludwig this should go in before the release.

File: web/vibe/web/web.d [hunk 1]
```diff
@@ -80,11 +80,19 @@ import std.encoding : sanitize;
...
 
 		$(UL
-			$(LI An array of values is mapped to
+			$(LI A dynamic array of values is mapped to
 				`<parameter_name>_<index>`, where `index`
-				denotes the zero based index of the array entry. The length
-				of the array is determined by searching for the first
-				non-existent index in the set of form fields.)
+				denotes the zero based index of the array entry. Any missing
+				indexes will be left as uninitialized. Arrays can also be
```
Comment (s-ludwig) [inline web/vibe/web/web.d]: Shouldn't this be "left to their `init` value" instead of "uninitialized"?

Comment (schveiguy) [reply web/vibe/web/web.d]: Well, I consider that uninitialized :) But yes, I can change.

Comment (schveiguy) [reply web/vibe/web/web.d]: fixed.

**Comment by s-ludwig** [comment]
Okay, thanks! Force-merging to bypass the CI delay. I'll just wait for #2265 before tagging the beta.

**Pull request merged**